### ISSUE TITLE
functional correctness question

### ISSUE THREAD

**Issue by alex-111-gh** [kunzmi/ImageStackAlignator#7]
Are there tests that verify functional correctness overall and of individual steps?

In other words, does it work? :)

**Comment by kunzmi** [comment]
Well, I wasn't aiming for industrial code quality when implementing :) 
So no, there are no unit tests, no tests suits for no regression or what so ever. All I wanted is to check the algorithm given my doubts due to the lack of quality in the paper… The existence of that GUI is, on the other hand, only to help me to get things right, to easily visualize the output of every single step, the different weights, the shifts, the tiles, and so on. 
You can manipulate basically all parameters that are used; that gives you a lot of power and allows you to mess things up. I spent a lot of time to ensure that it does actually work, if the parameters are right.
In short, I tested and debugged a lot, I'm certain that the individual functions do what I want them to do. But I can't ensure that what I want is correct as there's no reference implementation…

And the images I tested worked well, too.
